verifyFiles.py

```diff
@@ -147,13 +149,21 @@ def n_differences_across_subjects(conditions_dict,root_dir,metrics):
                     if(conditions_dict[c][subject][file_name].st_size != conditions_dict[d][subject][file_name].st_size):
                         #diff[key][file_name]["binary_content"]+=1
 			diff[key][file_name]+=1
-                        files_are_different=True
+                        files_are_different=True 
                     else:
-                        # File sizes are identical: compute the checksums
-                        abs_path_c=os.path.join(root_dir,c,subject,file_name)
+			abs_path_c=os.path.join(root_dir,c,subject,file_name)
                         abs_path_d=os.path.join(root_dir,d,subject,file_name)
-                        if checksum(abs_path_c) != checksum(abs_path_d): # TODO:when they are multiple conditions, we will compute checksums multiple times.
-                                                                         # We should avoid that.
+                        # File sizes are identical: compute the checksums
+	                if checksums_flag:
+                            folder_c_checksums_file=os.path.join(root_dir,c,subject,"checksums-after.txt")
+			    folder_d_checksums_file=os.path.join(root_dir,d,subject,"checksums-after.txt")
+			    #print folder_c_checksums_file,folder_d_checksums_file
+                            if ((os.path.isfile(folder_c_checksums_file) and os.path.isfile(folder_d_checksums_file))):
+				if not ((os.path.isdir(abs_path_c) and os.path.isdir(abs_path_d))):
+    			            if not is_checksum_equal(folder_c_checksums_file,folder_d_checksums_file,subject,file_name):
+				        diff[key][file_name]+=1
+                                        files_are_different=True                            
+                        elif checksum(abs_path_c) != checksum(abs_path_d): # TODO:when they are multiple conditions, we will compute checksums multiple times.We should avoid that.
```

Comment:
So here we recomputed the checksums and found that they are different than what had been computed in CBRAIN. It means that the file(s) were corrupted during transfer. This is a very serious error. We should print an error message and stop the script here (using `log_err`).